templates/about.html

```diff
@@ -1,6 +1,32 @@
 {% extends "layout.html" %}
 {% block content %}
 
-about
+<head>
+<title> ABOUT </title>
+<h1> About Mars Rocks </h1>
+
+<style>
+h2 {
+text-align: center;
+font-family: Impact;
+color: burgundy
+}
+
+</style>
+
+</head>
+
+<body>
+<h2><strong>Welcome to the Mars Rocks!</strong></h2>
+
+<img src=http://i1.wp.com/www.universetoday.com/wp-content/uploads/2008/06/mars_rocks.jpg?resize=246%2C249>
+
+<h3>Our webapp allows users to identify interesting features on Mars using the <a href=http://mars.jpl.nasa.gov/msl/mission/rover/” Mars rover> </a> images. </h3>
+
+<p> <p style=color: white; background-color: red; font-size: 20px”>
+NASA Mars rovers take tons of photos and NASA needs help from the public to look through the photos to identify interesting geological features in each photo. We will get the photos from NASA's api and allow users, access to these photos. The user is then allowed to comment on the photos. The app has a leaderboard that shows the total number of features that all the users have identified and another feature also lists the users who have identified the most photos.
```

Comment:
You have two <p> tags. Delete the second one. Neil is working on the styling so you don't need to include styles in the html.